src/modules/minigame/common/debug/debug_minigame_upgrades.gd

```diff
@@ -67,7 +65,7 @@ func _add_upgrade_item(upgrade: BaseUpgrade) -> void:
 
 
 func _get_upgrade_button_text(upgrade: BaseUpgrade) -> String:
-	return "%s Lvl %d/%d" % [upgrade.name, upgrade.get_level(), upgrade.get_max_level()]
+	return "%s Lvl %d/%d" % [upgrade.name, upgrade.get_level() + 1, upgrade.get_max_level() + 1]
```

Comment:
I think this +1 for max_level is wrong... 🤔 since it's already the length of the array, shouldn't it already be the correct amount?

the actual level makes sense, since it starts at -1 I think?

I'll approve for now anyway 👀  maybe I'm wrong